### ISSUE TITLE
DAP Violation on DataTableSkeleton button

### ISSUE THREAD

**Issue by nictownsend** [carbon-design-system/carbon#6026]
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships - SC 1.3.1

![image](https://user-images.githubusercontent.com/47386186/81180996-b3f76e80-8fa3-11ea-95af-c836ac7ecd64.png)

https://codesandbox.io/s/naughty-jang-qfv43

**Comment by nictownsend** [comment]
That was quick! Thank you :)
